### PR TITLE
feat - added wish-list page

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Used Card of component library
-> This is preview link - https://deploy-preview-5--music-mart-screens.netlify.app/wishlist-page/wishlist-page

### CODE REVIEW

**Review by AmishMishra11 — APPROVED** [review]
clean code good job.

File: wishlist-page/wishlist-page.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        </li>
+      </ul>
+    </div>
+    <div class="margin-req"></div>
+    <div class="container">
+      <div class="flex-center-row mar-t-1 pad-1 fnt-1-2 fnt-w-600">
+        MY WISHLIST
+      </div>
+      <div class="responsive-grid">
+        <div class="card vertical-card">
```
Comment (lannisterrr) [inline wishlist-page/wishlist-page.html:73]: wishlist card should have a remove icon/badge , so that user can remove it from the wishlist.

**Review by prathmeshjagtap — COMMENTED** [review]
we have to add a remove icon to remove the card from the wishlist maybe you can work on that , everything else looks good.

**Review by lannisterrr — COMMENTED** [review]
Other than points mentioned above , the code seems fine.

File: wishlist-page/wishlist-page.html (hunk 1 above)
Comment (ashishnnnnn) [reply wishlist-page/wishlist-page.html:73]: Yaa... forgot that before..
Added it ..

**Comment by ashishnnnnn** [comment]
Yes.. added that remove button

**Review by prathmeshjagtap — APPROVED** [review]

**Pull request merged**